src/templates/changetip.html

```diff
@@ -0,0 +1,29 @@
+{% extends "layout.html" %}
+
+{% block title %} Edit tip {% endblock %}
+
+{% block body %}
+
+<h2 class="mb-3">Edit tip</h2>
+<p class="mb-0">Current title: &nbsp {{tip.title}}</p>
+<p>Current link: &nbsp {{tip.link}}</p>
```

Comment:
It is likely often desired to only edit one of these at a time, so the form should be prefilled with the existing values.

Not sure if showing the original values separately is necessary, but it's not a bad idea in itself.